apps/web/src/pages/api/v1/admin/get-claims.ts

```diff
@@ -3,23 +3,24 @@ import { NextApiResponse } from 'next'
 
 import configureAdmin from '@/clients/firebase-admin-client'
 import createHandler, { NextApiRequestApp } from '@/middleware'
-import adminMiddleware from '@/middleware/admin-middleware'
 import authMiddleware from '@/middleware/auth-middleware'
 import userMiddleware from '@/middleware/user-middleware'
 
 const handler = createHandler()
-handler.use(authMiddleware()).use(userMiddleware()).use(adminMiddleware())
+handler.use(authMiddleware()).use(userMiddleware())
```

Comment:
Since this does not do anything but read the claims for a user, this should be safe to make public for authenticated users. This way, we don't throw a `Forbidden` error just for checking if the user is an admin.